.gitignore

```diff
@@ -1,3 +1,15 @@
+
```

Comment:
I've already seen that you leave the first line blank in your PRs. Can please clarify the purpose?

Reply:
Got it! If it's not critical for you, please remove it.

Reply:
yes definitely! I'll do that right now

Reply:
updated!

Reply:
Many thanks! Really appreciate your help here!

Will merge right after CI green check mark.